subsys/dfu/dfu_target/src/dfu_target_mcuboot.c

```diff
@@ -16,6 +16,7 @@
 #endif
 #include <string.h>
 
+#include <autoconf.h>
```

Comment:
This should not be needed. See https://github.com/zephyrproject-rtos/zephyr/pull/44471.

Reply:
Fixed